src/mgos_wifi.c

```diff
@@ -152,6 +153,14 @@ static void mgos_wifi_on_change_cb(void *arg) {
   }
 }
 
+void mgos_wifi_dev_ap_on_change_cb(enum mgos_wifi_event ev){
+  // struct mgos_wifi_ap_sta_connected_arg evd = {
+  //     //
+  // };
+  // mgos_event_trigger(ev, &evd);
+  mgos_event_trigger(ev, NULL); // not sure correct way to pass ev info from platform specific files
```

Comment:
i think it would be ok to trigger AP_STACONENCTED/DISCONNECTED ditrectly from platform files but you need to make sure it happens on the mgos task. `esp32_wifi_ev` runs on a different task, so you need to put it through `mgos_invoke_cb`.

Reply:
Passing a heap allocated structure to `mgos_invoke_cb` triggers a `Guru Meditation Error: Core  0 panic'ed (LoadProhibited)` in [mgos_task](https://github.com/cesanta/mongoose-os/blob/2b4cbd0ffb6d5865a8a3c29243faf32857800a98/fw/src/mgos_hal_freertos.c#L226)

ESP8266 triggers a similar error in [mgos_lwip_task](https://github.com/cesanta/mongoose-os/blob/2b4cbd0ffb6d5865a8a3c29243faf32857800a98/fw/platforms/esp8266/src/esp_main.c#L249)